content/yaml-publishing/google-play.md

```diff
@@ -86,6 +86,10 @@ publishing:
     # or your custom track name
     track: internal
 
+    # Optional text. The release name.
```

Comment:
seeing how we often specify the expected type of the value, then I'd go with "Optional string" here